input/parameters/lead-acid/experiments/1C_discharge_from_full/parameters.csv

```diff
@@ -10,8 +10,7 @@ Number of electrodes connected in parallel to make a cell,8,Manufacturer,
 Number of cells connected in series to make a battery,6,Manufacturer,
 Lower voltage cut-off [V],1.73,,(just under) 10.5V across 6-cell battery
 Upper voltage cut-off [V],2.44,,(just over) 14.5V across 6-cell battery
-C-rate,1,,
-Current function,[constant],,
+C-rate,0.1,,
```

Comment:
should this still be 1?

Reply:
one of the tests failed when it was 1C so I just lazily changed it so 0.1C, don't really care about lead-acid any more ...